fili-core/src/main/java/com/yahoo/bard/webservice/table/resolver/RequestQueryGranularityResolver.java

```diff
@@ -43,7 +43,7 @@ public Granularity resolveAcceptingGrain(DataApiRequest apiRequest, TemplateDrui
             return requestGranularity;
         }
 
-        if (requestGranularity.satisfiedBy(queryGrain)) {
+        if (queryGrain.satisfiedBy(requestGranularity)) {
```

Comment:
This changes the meaning of this code!  Why are you doing this here?

Reply:
I changed it back